test/gh_to_db_parser.test.js

```diff
@@ -0,0 +1,56 @@
+var test             = require("tape");
+var parseIssues      = require("../api/parse_issues.js");
+var issueArray       = require("../example_issues.json");
+
+var parsedIssue1 = {
```

Comment:
Please consider creating a `fixtures` directory to put in data like this and then require it. 

As sample data is something that often is reused within other tests best to keep it in one place for easy access.

Also, consider the readability. When you open a test file you expect to see tests and it is a bit confusing to see data first thing.